manifests/native.json

```diff
@@ -248,6 +248,134 @@
       "mdnPath": "Global_Objects/Array/values",
       "category": "native"
     },
+    {
+      "type": "native",
+      "moduleName": "math.acosh",
+      "nodeVersion": "0.10.0",
+      "replacement": "Math.acosh",
+      "mdnPath": "Global_Objects/Math/acosh",
+      "category": "native"
+    },
+    {
+      "type": "native",
+      "moduleName": "math.asinh",
+      "nodeVersion": "0.10.0",
+      "replacement": "Math.asinh",
+      "mdnPath": "Global_Objects/Math/asinh",
+      "category": "native"
+    },
+    {
+      "type": "native",
+      "moduleName": "math.atanh",
+      "nodeVersion": "0.10.0",
+      "replacement": "Math.atanh",
+      "mdnPath": "Global_Objects/Math/atanh",
+      "category": "native"
+    },
+    {
+      "type": "native",
+      "moduleName": "math.cbrt",
+      "nodeVersion": "0.10.0",
+      "replacement": "Math.cbrt",
+      "mdnPath": "Global_Objects/Math/cbrt",
+      "category": "native"
+    },
+    {
+      "type": "native",
+      "moduleName": "math.clz32",
+      "nodeVersion": "0.10.0",
+      "replacement": "Math.clz32",
+      "mdnPath": "Global_Objects/Math/clz32",
+      "category": "native"
+    },
+    {
+      "type": "native",
+      "moduleName": "math.cosh",
+      "nodeVersion": "0.10.0",
+      "replacement": "Math.cosh",
+      "mdnPath": "Global_Objects/Math/cosh",
+      "category": "native"
+    },
+    {
+      "type": "native",
+      "moduleName": "math.expm1",
+      "nodeVersion": "0.10.0",
+      "replacement": "Math.expm1",
+      "mdnPath": "Global_Objects/Math/expm1",
+      "category": "native"
+    },
+    {
+      "type": "native",
+      "moduleName": "math.f16round",
+      "nodeVersion": "0.10.0",
+      "replacement": "Math.f16round",
+      "mdnPath": "Global_Objects/Math/f16round",
+      "category": "native"
+    },
+    {
+      "type": "native",
+      "moduleName": "math.fround",
+      "nodeVersion": "0.10.0",
+      "replacement": "Math.fround",
+      "mdnPath": "Global_Objects/Math/fround",
+      "category": "native"
+    },
+    {
+      "type": "native",
+      "moduleName": "math.hypot",
+      "nodeVersion": "0.10.0",
+      "replacement": "Math.hypot",
+      "mdnPath": "Global_Objects/Math/hypot",
+      "category": "native"
+    },
+    {
+      "type": "native",
+      "moduleName": "math.imul",
+      "nodeVersion": "0.10.0",
+      "replacement": "Math.imul",
+      "mdnPath": "Global_Objects/Math/imul",
+      "category": "native"
+    },
+    {
+      "type": "native",
+      "moduleName": "math.log10",
+      "nodeVersion": "0.10.0",
+      "replacement": "Math.log10",
+      "mdnPath": "Global_Objects/Math/log10",
+      "category": "native"
+    },
+    {
+      "type": "native",
+      "moduleName": "math.log1p",
+      "nodeVersion": "0.10.0",
+      "replacement": "Math.log1p",
+      "mdnPath": "Global_Objects/Math/log1p",
+      "category": "native"
+    },
+    {
+      "type": "native",
+      "moduleName": "math.log2",
```

Comment:
this one doesn't seem to exist on npm 👀 

can you double check it before i merge? 🙏

Reply:
the ones that don't seem to exist (anymore?):

- hypot
- expm1
- cosh
- asinh
- log2

Reply:
Interesting, see https://github.com/es-shims/Math.log2

Reply:
i suppose they never got published

we could leave them out and re-introduce them if they ever get published possibly?

as someone could claim the name meanwhile with a lighter polyfill

Reply:
Sure.